crates/core_arch/src/acle/registers/aarch32.rs

```diff
@@ -1,4 +1,5 @@
 /// Application Program Status Register
 pub struct APSR;
 
+#[cfg(any(not(target_feature = thumb-mode), target_feature = v6t2))]
```

Comment:
You will need to add quotes around the features I believe, and `thumb-mode` probably needs to be added to https://github.com/rust-lang/rust/blob/c2ddf5a1dd54ebe18ffb794e096c28a4ed8e1d16/src/librustc_codegen_llvm/llvm_util.rs#L96

Reply:
oh no! that's take a whole other PR first :/